### DISCUSSION TITLE
Transform class `Affined` missing

### DISCUSSION THREAD

**Discussion by function2-llx** [Project-MONAI/MONAI#1703] [General]
Hi there. 

Most of the array transform classes have their corresponding dictionary versions. However, `Affine` seems not to be the case, class `Affined` cannot be found anywhere while only `RandAffined` can be found. Is it deliberately desinged this way?

**Comment by rijobro** [comment]
Good spot, thanks! I've created a feature request issue here: https://github.com/Project-MONAI/MONAI/issues/1704.

**Reply by rijobro** [reply]
Solved by @Nic-Ma with https://github.com/Project-MONAI/MONAI/pull/1709!

**Reply by function2-llx** [reply]
Thanks for your great work!
